design/adr/core/README.md

```diff
@@ -0,0 +1,7 @@
+# ADR Table of Contents
+
+| Name/Link                                          | Short Description                            |
+| -------------------------------------------------- | -------------------------------------------- |
+| [0001 MongoDB Clearance](0001-MongoDB-Clearance.md)| Add Clear DB functionaly inside `edgex-mongo` |                                           
+| [0002 Developers Scripts](0001-Developer-Scripts-TaggingAndCleanup.md) | Clearing and start tagging Developers scripts |
+
```

Comment:
We're tracking ADRs at global level.  This README.md should not be included.  Regardless, it's contents are not valid based on the existing content in the design/adr/core directory.

Reply:
fixed